goal_src/pc/progress-pc.gc

```diff
@@ -899,7 +900,11 @@
                   (load-level-text-files (-> *level-task-data* (-> obj display-level-index) text-group-index))
                   (set! (-> obj next-display-state) (progress-screen invalid))
                   )
+                 (((game-text-id camera-controls-original))
+                  ;; restore default camera controls
+                  (reset-original-camera *pc-settings*)
```

Comment:
Add a `(sound-play-by-name (static-sound-name "cursor-options") (new-sound-id) 1024 0 0 1 #t)` so there's an audio confirmation when you select the option. This should (but I'm not sure) correspond to that "ding" sound.